### PR TITLE
Feature/client management

### PR DESCRIPTION
**Pull request by KumbiraiShonhiwa**
## Pull Request

**Description:**

Added the table view of all the clients with details (Mock Data)
New Client: Interactions correctly with the Client Details Modal
Editing: Interactions with Edit Client Modal
Deleting: Interactions with Delete Client Modal
Filtering and Searching for Client implmeneted
Sorting based on the Headers implemented

**Related Issues:**

n/a

**Type of Change:**

- [ ] Bug Fix
- [x] New Feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Documentation Update

**Checklist:**

- [ ] My code follows the coding standards of this project.
- [ ] I have performed a self-review of my code.
- [ ] I have added/updated relevant documentation.
- [ ] My changes generate no new warnings or errors.
- [ ] I have tested my changes.

**Screenshots (if applicable):**

image

**Reviewers:**

- @BlockchainDlamini 
- @CodeHermez 

**Additional Notes:**

n/a

### CODE REVIEW

File: src/frontend/src/components/NavigationBar.vue
Comment (BlockchainDlamini) [inline src/frontend/src/components/NavigationBar.vue:1]: Please pull from dev, you're navbar code is outdated

Comment (KumbiraiShonhiwa) [reply src/frontend/src/components/NavigationBar.vue:1]: image
 says im up to date

Comment (BlockchainDlamini) [reply src/frontend/src/components/NavigationBar.vue:1]: check the navbar code in develop

Comment (BlockchainDlamini) [reply src/frontend/src/components/NavigationBar.vue:1]: it's a minor thing, just remove your comments

**Review by CodeHermez — APPROVED** [review]
Good

**Pull request merged**